### PR TITLE
fix misc. typos

### PR DESCRIPTION
**Pull request by stefaneicher**
a small contribution :-)

### CODE REVIEW

File: include/logging/log_msg2.h [hunk 1]
```diff
@@ -125,7 +125,7 @@ union log_msg2_generic {
  */
 enum z_log_msg2_mode {
 	/* Runtime mode is least efficient but supports all cases thus it is
-	 * threated as a fallback method when others cannot be used.
+	 * threaded as a fallback method when others cannot be used.
```
Comment (aunsbjerg) [inline include/logging/log_msg2.h:128]: Shouldn't this be "treated" and not "threaded"?

**Comment by aunsbjerg** [comment]
Thanks for the fixes. Before this PR can be merged, you'll need to add a proper commit message and sign the commit. I suggest you read the [contribution guidelines](https://docs.zephyrproject.org/latest/contribute/index.html), they should explain everything :)

File: include/logging/log_msg2.h (hunk 1 above)
Comment (ceolin) [reply include/logging/log_msg2.h:128]: :+1:

**Review by ceolin — APPROVED** [review]
Other than the comment looks good to me.

File: include/logging/log_msg2.h (hunk 1 above)
Comment (stefaneicher) [reply include/logging/log_msg2.h:128]: Thanks, I will change that and to a squash commit

**Comment by stefaneicher** [comment]
I updated the commit message

**Review by chen-png — APPROVED** [review]

**Review by nashif — APPROVED** [review]

**Comment by enjiamai** [comment]
Hi, it did not pass the compliance check due to the author's email did not match the sign-up email. I saw a  
Signed-off-by: Stefan Eicher <stefan.eicher@gmail.com> in your commit. Could you please have an update? thanks!

```
Error: See https://docs.zephyrproject.org/latest/contribute/#commit-guidelines for more details

f7f414bdba811ad8f91a10ac0c45a9056c1769db: author email (Stefan Eicher <stefan.eicher@ypsomed.com>) needs to match one of the signed-off-by entries.
Error: Process completed with exit code 1.
```

**Pull request merged**